content/developer/rom/global/questionnaires.md

```diff
@@ -49,6 +51,19 @@ status: draft
       tot:  {label: "Somscore"},
       gedr: {label: "Problemen"},
       bep:  {label: "Dingen"}
+    },
+
+    sbg_info: {
+      key: "example_sbg_key",
+      questions: {
+        example_sbg_question_key_1: "example_1",
```

Comment:
Moet juist ver van tevoren aangekondigd en duidelijk in de documentatie staan. Anders kan je het net zo goed nu al aanpassen.

plus geeft richting van de api aan. 
wat je moet doen als het nu implementeerd.